src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -198,34 +215,15 @@ void ScatterUpdate::initSupportedPrimitiveDescriptors() {
     dataPrec = getOriginalInputPrecisionAtPort(DATA_ID);
     dataSize = dataPrec.size();
 
-    bool canBeInplace = !isDynamicNode() && getParentEdgeAt(DATA_ID)->getParent()->getChildEdges().size() == 1 &&
-                        !getParentEdgeAt(DATA_ID)->getParent()->isConstant();
-
-    NodeConfig config;
-    if (axisRelaxed) {
-        config.inConfs.resize(4);
-    } else {
-        config.inConfs.resize(3);
-    }
-    config.outConfs.resize(1);
-    config.inConfs[DATA_ID].constant(false);
-    config.inConfs[INDICES_ID].constant(false);
-    config.inConfs[UPDATE_ID].constant(false);
-    config.outConfs[0].constant(false);
-    config.inConfs[DATA_ID].inPlace(canBeInplace ? 0 : -1);
-    config.inConfs[INDICES_ID].inPlace(-1);
-    config.inConfs[UPDATE_ID].inPlace(-1);
-    config.outConfs[0].inPlace(canBeInplace ? 0 : -1);
-    if (axisRelaxed) {
-        config.inConfs[AXIS_ID].constant(false);
-        config.inConfs[AXIS_ID].inPlace(-1);
-    }
+    bool canBeInplace = !getParentEdgeAt(DATA_ID)->getParent()->isConstant();
 
-    std::vector<PortConfigurator> inPortConfig{{LayoutType::ncsp, dataPrec}, {LayoutType::ncsp, indicesPrec}, {LayoutType::ncsp, dataPrec}};
+    std::vector<PortConfigurator> inPortConfig{{LayoutType::ncsp, dataPrec, false, canBeInplace ? 0 : -1},
```

Comment:
Since now SEU impl is not just memcpy, we need to check that dataPrec equals to one of supported precisions (f32,i32,bf16,f16) and fallback on f32 otherwise.

Reply:
done